pandas/tests/frame/methods/test_replace.py

```diff
@@ -1356,3 +1356,11 @@ def test_replace_replacer_dtype(self, replacer):
         result = df.replace({"a": replacer, "b": replacer})
         expected = pd.DataFrame([replacer])
         tm.assert_frame_equal(result, expected)
+
+    def test_replace_after_convert_dtypes(self):
+        # GH31517
+        df = pd.DataFrame({"grp": [1, 2, 3, 4, 5]})
```

Comment:
I would just specify it here as `dtype="Int64"` instead of using `convert_dtypes` (in the end, the issue was that replace didn't work with Int64 data, the convert_dtypes was just a way to get such dtype)

Reply:
oh yeah, i was copying-pasting the example in issue description, sorry for my laziness. 😅 changed!